python/ray/serve/tests/test_metrics.py

```diff
@@ -274,13 +353,32 @@ def f(*args):
     assert num_deployment_errors[0]["application"] == "app"
     print("serve_num_deployment_http_error_requests working as expected.")
 
+    num_deployment_errors = get_metric_dictionaries(
+        "serve_num_deployment_grpc_error_requests"
+    )
+    assert len(num_deployment_errors) == 1
+    assert num_deployment_errors[0]["deployment"] == "app_f"
```

Comment:
You may have to merge master, the PR to remove application prefix was merged recently.

Reply:
Yea, I was expecting to rebase again. At the time when I wrote this, that change is not in master, let me double check